content/millennium/r4/financial/charge-item.md

```diff
@@ -83,23 +83,28 @@ Search for ChargeItems that meet supplied query parameters:
 
 ### Parameters
 
- Name                  | Required?  | Type          | Description
------------------------|------------|---------------|-------------------------------------------------------
- `_id`                 | Yes        | [`token`]     | The logical resource id associated with the ChargeItem.
+ Name                  | Required?       | Type          | Description
+-----------------------|-----------------|---------------|-------------------------------------------------------
+ `_id`                 | This or context | [`token`]     | The logical resource id associated with the ChargeItem.
+ `context`             | This or _id     | [`token`]     | Encounter associated with event.
+ `-status`             | See notes       | [`token`]     | The status of the ChargeItem. Example: `billable`
+ [`_count`]            | No              | [`number`]    | The maximum number of results to return.
 
 Notes:
 
 - When searching by `_id`, only a single ChargeItem id can be provided.
+- `-status` must be provided with context.
```

Comment:
put context in the same format as status

`context`

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/665/commits/7dcf1473cb198a27c98dcb618252085c6dcfb703#diff-16d4f8f74626c62d57a8b71d6be699f60e72fffed8636bc32e6654d4252b3ebbR96